src/d3a/models/strategy/area_agents/two_sided_pay_as_bid_engine.py

```diff
@@ -40,19 +44,19 @@ def _forward_bid(self, bid):
             return
         if self.owner.name == self.markets.target.area.name:
             return
-        if (bid.price * (1 - self.markets.target.transfer_fee_ratio)
-                - self.markets.target.transfer_fee_const * bid.energy) <= 0:
-            self.owner.log.debug("Bid is not forwarded because bid price lower "
-                                 "than transfer_fee_const")
-            return
+        # if (bid.price * (1 - self.markets.target.transfer_fee_ratio)
```

Comment:
Yeah, this shouldn't be needed any more, right? Are you keeping it just in case?

Reply:
You are right, this should be removed, deleting this